canonical-web-and-design/ubuntu.com
# Update case study on /security/esm

## Done

- Update case study on /security/esm
- Moved /security/esm and /security/livepatch to /security root and removed base files and directories

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/security/esm
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that the IT Strategen case study was replaced by the TIM one per the [copy doc](https://docs.google.com/document/d/1RBZX3PJPvODxRlcnIS10Ii4SkIMH5Fm74EmsMQ0cnic/edit?ts=5f6371c8#) and approve the suggestions
- See that I moved esm and livepatch to files in the /security folder and not as subfolders


## Issue / Card

Fixes #8314

## Screenshots

![image](https://user-images.githubusercontent.com/441217/93610389-e76fee00-f9c4-11ea-8633-8150521a1775.png)
